frontend/apps/reader/modules/readerfont.lua

```diff
@@ -231,4 +258,26 @@ function ReaderFont:addToMainMenu(menu_items)
     }
 end
 
+function ReaderFont:onPinch()
+    local info = Notification:new{text = _("Changing font size")}
+    UIManager:show(info)
+    UIManager:forceRePaint()
+    UIManager:nextTick(function()
+        self:onChangeSize("decrease")
+        UIManager:close(info)
+    end)
+    return true
+end
+
+function ReaderFont:onSpread()
+    local info = Notification:new{text = _("Changing font size")}
+    UIManager:show(info)
+    UIManager:forceRePaint()
```

Comment:
Without `forceRePatin` "Changing font size" isn't show.

This code won't work to:
    
```
    local info = Notification:new{text = _("Changing font size"), timeout = 1.0}
    UIManager:show(info)
    self:onChangeSize("increase")

```